build/rsi/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+FROM python:3.11.4-slim as base
+
+ARG ENV
+ARG PIP_DISABLE_PIP_VERSION_CHECK=1
+ARG PIP_NO_CACHE_DIR=1
+ENV ENV=${ENV}
+
+WORKDIR /workspace
+
+COPY simple/ .
```

Comment:
now under `import` repo there are `simple` and `rsi(really simple import` concepts. Might consider renaming folders later?

Reply:
How about calling it "lite", "dclite", or something similar instead of an acronym that has a more jargon feel?

Reply:
Good callouts.

> now under `import` repo there are `simple` and `rsi`

Any thoughts on reusing the "simple" and calling this `simple` or `simple_import`?

Reply:
Given the proposal in the other comment, reusing "simple" is simple.

Reply:
Done.